From b02a6df2dd9d34717c9bf8a8ab75325380216459 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Fri, 27 Nov 2015 13:16:06 +0100
Subject: [PATCH] mount.Info -> mount.MountInfo

---
 libcontainer/rootfs_linux.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcontainer/rootfs_linux.go b/libcontainer/rootfs_linux.go
index 5a2fad8..e4377cd 100644
--- a/libcontainer/rootfs_linux.go
+++ b/libcontainer/rootfs_linux.go
@@ -426,7 +426,7 @@ func mknodDevice(dest string, node *configs.Device) error {
 	return syscall.Chown(dest, int(node.Uid), int(node.Gid))
 }
 
-func getMountInfo(mountinfo []*mount.Info, dir string) *mount.Info {
+func getMountInfo(mountinfo []*mount.MountInfo, dir string) *mount.MountInfo {
 	for _, m := range mountinfo {
 		if m.Mountpoint == dir {
 			return m
-- 
1.9.3

